lib/code-caller/code-caller-native.js

```diff
@@ -44,61 +43,7 @@ const EXITED = Symbol('EXITED');
 
 /** @typedef {Error & { data?: ErrorData }} CodeCallerError */
 
-/*
-  Usage
```

Comment:
This information was all out of date, and (IMO) is no longer super useful. Happy to revert this and edit it to be up to date if that's preferred.

Reply:
I agree it doesn't seem necessary anymore.